contract/proxy-oracle/soroban/RUNBOOK.md

```
@@ -0,0 +1,1281 @@
+  --wasm target/proxy-oracle-soroban/wasm/templar_proxy_oracle_soroban_contract.optimized.wasm
+
+# Upgrade the deployed contract to the new WASM hash (governance must authorize)
+stellar contract invoke \
+  --network <network> \
+  --source <governance-identity> \
+  --id <RUNTIME_CONTRACT_ID> \
+  -- upgrade \
+  --new_wasm_hash <NEW_WASM_HASH>
+```
+
+2. **Governed proposal upgrade** (AdminUpgrade proposal, executes after maturity):
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)